core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.directive.html

```diff
@@ -36,7 +36,7 @@
     <div class="oppia-translation-editor-section">
       <div class="protractor-test-state-translation-editor">
         <ck-editor-copy-toolbar></ck-editor-copy-toolbar>
-        <schema-based-editor schema="HTML_SCHEMA" local-value="activeWrittenTranslation.html">
+        <schema-based-editor schema="HTML_SCHEMA" local-value="activeWrittenTranslation.html" dir="<[targetLocaleDirection]>">
```

Comment:
As you have added direction for the "Content" language should we add the direction in all the editor (an entity which can have content like exploration/question etc)? (This seems like out of scope of this PR, you can create a Github issue for now and let's fix it later.)

I think we should have langaugeCode in the `schema` attribute and the editor directive should decide the direction? (not sure! Though this way we won't have to fetch the direction of a language in all the parent directive i.e, all the parent directive will just pass the languageCode and the editor-directive will have code to find the direction.)

Can you also add this in the translation tab of the exploration editor page? (As of now we also have another place (Exploration editor page) from where someone can add a translation.)

Reply:
I think this is a good idea, I'll look into adding it to the schema
and I'll make an issue for adding it to the translation tab of the exploration editor page

Reply:
In this PR lets add direction in all the places where we can add translation i.e, (translation-tab of the exploration-editor page and contributor-dashboard)  [I think we only need a few minor changes but feel free to defer it for later if it's complicated] and file an issue to add direction in all the places where we are using the editor (e.g, exploration hint, solution, questions etc.) as per the entity language.

Reply:
The latest change works for all translation text editors.

Reply:
This is now uses the schema attribute as you suggested.